start.sh

```diff
@@ -10,11 +11,13 @@ if [ "$SSH_PUBLIC_KEY" ]; then
 fi
 
 # create an empty repo
+echo 'creating repo...'
 cd /home/git
 su git -c "git init  --bare -b main repo"
 
 # use the repo files (if any) to make an initial commit in the repo
 if [ "$(ls -A /root/repoFiles/)" ]; then
```

Comment:
I'm not a fan of this line as it outputs an error if the folder doesn't exist, but that's fine since this is an `if` statement. I'd use the `-d` operator to check for directory existence instead:

```suggestion
if [ -d /root/repoFiles ]; then
```